hcloud/provider.go

```diff
@@ -68,6 +68,13 @@ func Provider() *schema.Provider {
 				Default:     "500ms",
 				Description: "The interval at which actions are polled by the client. Default `500ms`. Increase this interval if you run into rate limiting errors.",
 			},
+			"poll_function": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "exponential",
+				Description:  "The type of function to be used during the polling.",
+				ExactlyOneOf: []string{"constant", "exponential"},
```

Comment:
```suggestion
				ValidateFunc: validation.StringInSlice([]string{"constant", "exponential"}, false),
```

Sorry for this late find, `ExactlyOneOf` is used to tell terraform that one of the specified fields must be set, not that this field needs to be one of two values.

Reply:
Got it! Thanks @apricote